policyengine_uk/parameters/gov/dfe/targeted_childcare_entitlement/qualifying_benefits.yaml

```diff
@@ -0,0 +1,24 @@
+description: The Department for Education designates these programs as qualifying benefits for the targeted childcare entitlement eligibility (the 2-year-old entitlement) in England.
+metadata:
+  reference:
+    - title: Local Authority (Duty to Secure Early Years Provision Free of Charge) Regulations 2014 - Regulation 3 defining "eligible child"
+      href: https://www.legislation.gov.uk/uksi/2014/2147/regulation/3/made
+    - title: Education Act 1996 s.512ZB(4) - criteria for free school meals
+      href: https://www.legislation.gov.uk/ukpga/1996/56/section/512ZB
+    - title: Local Authority (Duty to Secure Early Years Provision Free of Charge) (Amendment) Regulations 2018
+      href: https://www.legislation.gov.uk/uksi/2018/146/contents/made
+    - title: GOV.UK - Free education and childcare for 2-year-olds
+      href: https://www.gov.uk/help-with-childcare-costs/free-childcare-2-year-olds
```

Comment:
I don't see these benefits in the law but they are on gov.uk. Could you find them?